react/features/gifs/reducer.ts

```diff
@@ -23,6 +25,7 @@ export interface IGifsState {
     drawerVisible: boolean;
     gifList: Map<string, IGif>;
     menuOpen: boolean;
+    overflowVisible: boolean;
```

Comment:
Wouldn't it make more sense to turn these 2 variables into a `mode` enum? The mode would be either `null`, `drawer` or `overflow`.

Reply:
Good idea! Will give it a try!

Reply:
This is related to the previous reply. Now `overflowVisible` and `drawerVisible` are gone because we don't need them anymore!